### ISSUE TITLE
When will RAWCooked move from alpha to production ready?

### ISSUE THREAD

**Issue by kieranjol** [MediaArea/RAWcooked#142]
What are the milestones for moving out of alpha, and what conditions are to be met for this tool being 'production ready'?
On our end, I think I'd be satisfied with having a workflow/script in place that performs the normalisation and then performs a losslessness check. I'm curious to know what others think.

**Comment by JeromeMartinez** [comment]
Milestone was "April", obviously it was not here in April.
Not an excuse, I got some unexpected work on other tools, and I did not work enough on RAWcooked but I plan to have a lot of time for RAWooked in September, so my updated goal is to move to "production ready" before [No Time To Wait 3](http://mediaarea.net/nttw3).

Conditions:
- implementation of the licensing part (restrict binaries to 8/10-bit DPX and 16-bit WAV if no license) in order to motivate new sponsors.
- implementation of some TIFF and AIFF support in order to be sure it is well designed for handling several input formats.
- implementation of the support of reels in order to be sure it is well designed for handling "concatenated" WAV in the MKV/FFV1/FLAC and during revert.
- Code cleanup

Also, users need to test, test and test ;-).
